src/plugins/controls/server/options_list/options_list_queries.ts

```diff
@@ -26,12 +26,27 @@ interface EsBucket {
  * Validation aggregations
  */
 export const getValidationAggregationBuilder: () => OptionsListAggregationBuilder = () => ({
-  buildAggregation: ({ selectedOptions, fieldName }: OptionsListRequestBody) => {
-    const selectedOptionsFilters = selectedOptions?.reduce((acc, currentOption) => {
-      acc[currentOption] = { match: { [fieldName]: currentOption } };
-      return acc;
-    }, {} as { [key: string]: { match: { [key: string]: string } } });
-
+  buildAggregation: ({
+    selectedOptions,
+    fieldName,
+    existsSelected,
+    exclude,
+  }: OptionsListRequestBody) => {
+    let selectedOptionsFilters;
+    if (existsSelected) {
+      if (exclude) {
+        selectedOptionsFilters = {
+          existsQuery: { bool: { must_not: { exists: { field: fieldName } } } },
+        };
+      } else {
+        selectedOptionsFilters = { existsQuery: { exists: { field: fieldName } } };
+      }
+    } else if (selectedOptions) {
+      selectedOptionsFilters = selectedOptions.reduce((acc, currentOption) => {
+        acc[currentOption] = { match: { [fieldName]: currentOption } };
+        return acc;
+      }, {} as { [key: string]: { match: { [key: string]: string } } });
+    }
```

Comment:
We could actually expand this to have `Not <a, b, c>` selections validate differently than `<a, b, c>` - however, when I added this, while the behaviour was **technically** correct it was, in my opinion, more confusing for users than simply validating the same way regardless of whether or not `exclude` is true. 

For example, consider the following scenario:

If we validated the `Not` selections differently, then the second control would actually be **valid** because

Reply:
**Edit:** This no longer applies since validation has been removed.